docs/proposals/volume-unification.md

```diff
@@ -0,0 +1,65 @@
+## Abstract
+
+A proposal for enabling several Kubernetes resources to be available
+within the same volume mount.
+
+## Motivation
+
+
+## Constraints and Assumptions
+
+It is assumed that any specified paths do not cause overlap with other
```

Comment:
This needs to be more general, and I doubt that this is a valid constraint.  What does 'error' mean here?